requirements_test.txt

```diff
@@ -1,4 +1,3 @@
--r requirements_test_pre_commit.txt
```

Comment:
I wonder if we should update the last paragraph [here](https://pylint.readthedocs.io/en/latest/development_guide/contributor_guide/tests/install.html#contributor-installation) about pre-commit installation to point out the need to install the pre-commit requirements?